### DISCUSSION TITLE
Hilfe

### DISCUSSION THREAD

**Discussion by Paulbaumann1** [community/community#34076] [Other Feature Feedback, Questions, & Ideas]
Hab keine Ahnung von github mir aber ein Account geholt um irgendwie Kontakt zu Klaus Ahrensberg zu bekommen
Suche dringend ein Raumthermostat was ich entclouden kann mit tyua-konvert
Die 8 angegebenen sind alle Unterputz!
Kann mich natürlich irren...
Ich brauche ein Aufputz-Gerät!!
Mit tyua-konvert habe ich ja schon Erfahrung da denke ich das ich die Ahrensberg Firmware schon Rauf bekomme, mir fehlt nur ein Aufputz-Thermostat kann mir geholfen werden??

**Comment by airtower-luna** [comment]
Ich fürchte das ist hier eher nicht der richtige Ort, hier geht es hauptsächlich um Git und GitHub.

Kann natürlich sein daß jemand hier zufällig solche Geräte kennt, aber das wäre schon ein recht großer Zufall. Falls du mit "tuya-konvert" https://github.com/ct-Open-Source/tuya-convert meinst probier es doch mal unter "Discussions" da.

**Comment by Paulbaumann1** [comment]
Ja ich habe schon Schwierigkeiten überhaupt wieder in diesen "Diskussionen" zu kommen Github ist wohl nicht mein Ding abernur hier hat der Klaus Ahrenbergunter Replaces original Tuya firmware on Beca thermostat with ESP8266 wifi module.Was eingestellt und ist da auch unter klausahrenbergverzeichnetIch weiß bloß nicht wie ich in Kontakt mit ihm treten kann, wie kann ich ein Github - Member ansprechen?? Er verzeichnet 8 Unterputzdosen als kompatibel ich benötigte aber ein Raumthermostat was aufputz ist? Hat jemand eine Idee wie ich einen github user eine Frage stellen kann, in dem Falle den Nutzer klaus ahrenberg? Von meinem Telefon gesendet-------- Ursprüngliche Nachricht --------Von: Airtower ***@***.***>Datum: Sa., 24. Sept. 2022, 21:36An: community/community ***@***.***>Cc: Paulbaumann1 ***@***.***>, Author ***@***.***>Betreff: Re: [community/community] Hilfe (Discussion #34076)

Ich fürchte das ist hier eher nicht der richtige Ort, hier geht es hauptsächlich um Git und GitHub.

Kann natürlich sein daß jemand hier zufällig solche Geräte kennt, aber das wäre schon ein recht großer Zufall. Falls du mit "tuya-konvert" https://github.com/ct-Open-Source/tuya-convert meinst probier es doch mal unter "Discussions" da.

**Comment by airtower-luna** [comment]
Es gibt keine DMs oder sowas, manche Leute haben ein Mailaddresse im Profil. Bei der Art Frage wäre es aber unhöflich und auch nicht sinnvoll, direkt an jemanden zu schreiben. Besser allgemein an Leute, die die betreffende Software nutzen. Daher ja mein Hinweis auf das Repository.

**Comment by Paulbaumann1** [comment]
Danke aber ich bin zu blöd den Text in dem entsprechenden repro zu platzieren, ich komme mit der Nutzerführung nicht zurecht Irgendwie muss man doch demjenigen der die Kompatibelitätsliste hinterlegt hat eine Frage stellen können? Von meinem Telefon gesendet-------- Ursprüngliche Nachricht --------Von: Airtower ***@***.***>Datum: So., 25. Sept. 2022, 21:37An: community/community ***@***.***>Cc: Paulbaumann1 ***@***.***>, Author ***@***.***>Betreff: Re: [community/community] Hilfe (Discussion #34076)

Es gibt keine DMs oder sowas, manche Leute haben ein Mailaddresse im Profil. Bei der Art Frage wäre es aber unhöflich und auch nicht sinnvoll, direkt an jemanden zu schreiben. Besser allgemein an Leute, die die betreffende Software nutzen. Daher ja mein Hinweis auf das Repository.
